package.json

```diff
@@ -3,6 +3,7 @@
   "private": true,
   "version": "0.1.0",
   "dependencies": {
+    "@datanerd/tessen": "^1.3.0",
```

Comment:
This is going to cause issues for anybody not on the New Relic VPN (all non-employee contributors) since `@datanerd` does not live in NPM. How do we best approach this so that we can still have external contributors to the project?